db/migrate/20240327043323_set_admin_sidebar_enabled_groups.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class SetAdminSidebarEnabledGroups < ActiveRecord::Migration[7.0]
```

Comment:
This is **extremely** nitpicky but I think a slightly better migration name would be `DisableAdminSidebarForExistingSites` or something